extended/jams2015.tex

```diff
@@ -275,25 +276,26 @@ \subsection{Task namespaces}\label{sec:schema:namespace}
 
 With the namespace abstraction, it is possible for observations to have arbitrarily structured value and confidence fields.
 \Cref{fig:thayer} provides a complete example namespace definition for \emph{mood\_thayer} annotations, in which each observed value is an ordered pair of numbers encoding \emph{valence}
-and \emph{arousal} in the Thayer mood model.
+and \emph{arousal} in the Thayer mood model~\cite{thayer}.
 
 For convenience, namespaces are grouped into high-level task categories by their identifiers.
 We stress that this grouping is merely cosmetic, and there is no strict underlying hierarchy of tasks.
-\Cref{tab:namespaces} lists the namespaces supported in JAMS v0.2.0.
+\Cref{tab:namespaces} lists the namespaces supported in JAMS v0.2.
 
 Finally, namespaces are defined externally to the core schema, and new namespaces can be imported dynamically with no modifications to the JAMS implementation itself.
 This makes it possible to develop and share custom annotation specifications.
 
 \begin{table}
-    \caption{Namespaces supported in JAMS v0.2.0.}\label{tab:namespaces}
+    \caption{Namespaces supported in JAMS v0.2.}\label{tab:namespaces}
     \centering
     \begin{tabular}{rl}
     \toprule
     Task group                  & Namespace\\
     \midrule
-    \multirow{2}{*}{Beat}       & \texttt{beat}\\
+    Beat       & \texttt{beat}\\
```

Comment:
This is just a preference thing, but I find it to be clearer/more readable this way. When they were in the middle it was unclear to me which group was associated with which rows in the right column.